armonik/variables.tf

```diff
@@ -325,71 +263,80 @@ variable "authentication" {
   }
 }
 
-# The name of the secrets.
-variable "fluent_bit_secret_name" {
-  description = "the name of the fluent-bit secret"
-  type        = string
-  default     = "fluent-bit"
-}
-
-variable "grafana_secret_name" {
-  description = "the name of the grafana secret"
-  type        = string
-  default     = "grafana"
-}
-
-variable "prometheus_secret_name" {
-  description = "the name of the prometheus secret"
-  type        = string
-  default     = "prometheus"
-}
-
-variable "metrics_exporter_secret_name" {
-  description = "the name of the metrics exporter secret"
-  type        = string
-  default     = "metrics-exporter"
-}
-
-variable "partition_metrics_exporter_secret_name" {
-  description = "the name of the partition metrics exporter secret"
-  type        = string
-  default     = "partition-metrics-exporter"
+# The output of modules.
+variable "fluent_bit_output" {
```

Comment:
I am not sure about the output in the name of this variable. I get why you have put that, but I think it would be better to just name it `fluent_bit`